include/mata/inter-aut.hh

```diff
@@ -1,3 +1,4 @@
+//TODO: rename to intermediate_afa / afa-intermediate, or something like that.
```

Comment:
It actually is a kind of afa. Anyway, I somehow don't like those abbreviated names. 

Another thing about names of files, why do we have the prefix nfa or afa? They could be dropped no?

Reply:
Yes, it is a kind of afa because nfa is kind of afa. However, a spirit of this class is to be an intermediate representation between parser and efficient automata data structures. Therefore relating it to a certain kind of automata would be confusing.